src/main/java/seedu/exercise/logic/parser/ParserUtil.java

```diff
@@ -143,7 +147,7 @@ public static Date parseEndDate(Date startDate, String endDate) throws ParseExce
         String trimmedEndDate = endDate.trim();
         parseDate(trimmedEndDate);
         if (!Date.isEndDateAfterStartDate(startDate.toString(), trimmedEndDate)) {
-            throw new ParseException("End date must be before ");
+            throw new ParseException("End date must be after start date");
```

Comment:
Would be better if the message is stored in a final String MESSAGE_INVALID_END_DATE in the ParserUtil class or Date class